<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>308</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>446</width>
    <height>308</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>446</width>
    <height>308</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Моё детство</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../images/logo.png</normaloff>../images/logo.png</iconset>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>42</y>
      <width>131</width>
      <height>191</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Мультики</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Рисование</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Игры с соседями</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Конструкторы</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Компьютер</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Головоломки</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Книжки</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_task">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>6</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Любимое занятие в детстве:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_img">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>281</width>
      <height>191</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>191</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>191</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="pixmap">
     <pixmap>../images/childhood.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_selected">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>240</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Выбрано:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_back">
    <property name="geometry">
     <rect>
      <x>109</x>
      <y>270</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Назад</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_next">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>270</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Вперёд</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
